packages/permission-controller/src/PermissionController.ts

```diff
@@ -853,6 +855,25 @@ export class PermissionController<
       throw failureError;
     }
 
+    if (specification.subjectTypes) {
```

Comment:
```suggestion
    if (specification.subjectTypes?.length > 0) {
```

Reply:
`'specification.subjectTypes.length' is possibly 'undefined'`

Reply:
Should be addressed now